build/parameters.cake

```diff
@@ -111,7 +111,7 @@ public class BuildParameters
             IsReleaseBuild = IsReleasing(target),
             SkipSigning = StringComparer.OrdinalIgnoreCase.Equals("True", context.Argument("skipsigning", "False")),
             SkipGitVersion = StringComparer.OrdinalIgnoreCase.Equals("True", context.EnvironmentVariable("CAKE_SKIP_GITVERSION")),
-            SkipOpenCover = StringComparer.OrdinalIgnoreCase.Equals("True", context.EnvironmentVariable("CAKE_SKIP_OPENCOVER"))
+            SkipOpenCover = true//StringComparer.OrdinalIgnoreCase.Equals("True", context.EnvironmentVariable("CAKE_SKIP_OPENCOVER"))
```

Comment:
@patriksvensson @gep13 why is it disabled? Doesn't it work with the new tooling? If so issue would be great describing why.